Cargo.toml

```diff
@@ -20,12 +20,12 @@ spatial_index = ["dep:rstar"]
 
 [dependencies]
 ahash = "0.8"
-arrow2 = { version = "0.17", features = [] }
-geo = "0.26"
-geoarrow = { package = "geoarrow2", version = "0.0.1", optional = true, features = ["geozero"] }
+arrow = "49.0.0"
```

Comment:
arrow is up to 50 now

Reply:
Yep, the 49 was due to me using geoarrow2 0.0.2